### PR TITLE
Simplify the logic associated binary Support Files

### PR DESCRIPTION
**Pull request by vancem**
There are two nuget packages that we used to carry binary files that are needed
by perfview but are not built by this repo.    There was some copy logic in those
that was causing grief for Visual Studio.   Simplify this to avoid the need for the
copy.

Upgraded these support nuget packages to version 1.0.2.   This is mostly a no-op (the binaries did not change, the the possible exception of small changes to OSExtentions.dll)
but I needed to update these nuget packages to simplify them.

Also unbroke the logic for generating these support files.

Finally removed some .SLN files that have not been used in a long time and are probably confusing.

### CODE REVIEW

**Comment by vancem** [comment]
This fixes issue #198

File: src/NugetSupportFiles/Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles.Populate.bat [hunk 1]
```diff
@@ -1,5 +1,5 @@
 REM copies from an existing build to a nuget package creation area (so that *.MakeNuget.bat works)
-xcopy /s ..\PerfView\packages\Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles.1.0.1\*.dll Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles
+xcopy /s ..\..\packages\Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles.1.0.1\*.dll Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles
```
Comment (sharwell) [inline src/NugetSupportFiles/Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles.Populate.bat]: ❓ Should this be 1.0.**2**?

File: src/NugetSupportFiles/PerfView.SupportFiles.Populate.bat [hunk 2]
```diff
@@ -1,6 +1,6 @@
 REM copies from an existing build to a nuget package creation area (so that *.MakeNuget.bat works)
-xcopy /s /y ..\PerfView\packages\PerfView.SupportFiles.1.0.1\*.dll PerfView.SupportFiles
-xcopy /s /y ..\PerfView\packages\PerfView.SupportFiles.1.0.1\*.exe PerfView.SupportFiles
+xcopy /s /y ..\..\packages\PerfView.SupportFiles.1.0.1\*.dll PerfView.SupportFiles
```
Comment (sharwell) [inline src/NugetSupportFiles/PerfView.SupportFiles.Populate.bat]: :memo: Same question regarding 1.0.**2**.

File: src/NugetSupportFiles/Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles.Populate.bat (hunk 1 above)
Comment (vancem) [inline src/NugetSupportFiles/Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles.Populate.bat]: Well, at the time this was used, there did not exist a 1.0.2 to copy from (that was what was being created).    Still I can update it now, and I added some more comments / docs to make it less likely people mess this up.

**Comment by vancem** [comment]
The following test fails  
  LinuxTracing.Tests.EventParseTests.OneStack [FAIL]

Because of a file-in-use error 

      System.IO.IOException : The process cannot access the file 'C:\projects\perfview\src\LinuxEvent.Tests\bin\Release\Sources\onegeneric.perf.data.dump' because it is being used by another process.
      Stack Trace:
           at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
           at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
           at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
           at System.IO.FileStream..ctor(String path, FileMode mode)
        src\PerfView\OtherSources\Linux\LinuxPerfScriptStackSource.cs(414,0): at Diagnostics.Tracing.StackSources.LinuxPerfScriptStackSource.GetPerfScriptStream(String path, ZipArchive& archive)
---------
Which seems to be unrelated to the checkin.   Should followup on this error eventually.

**Pull request merged**